org.osgi.test.assertj.framework/src/main/java/org/osgi/test/assertj/bundle/BundleConditions.java

```diff
@@ -0,0 +1,63 @@
+/*******************************************************************************
+ * Copyright (c) Contributors to the Eclipse Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *******************************************************************************/
+
+package org.osgi.test.assertj.bundle;
+
+import java.util.Objects;
+
+import org.assertj.core.api.Condition;
+import org.assertj.core.api.ListAssert;
+import org.assertj.core.api.ObjectAssert;
+import org.assertj.core.condition.VerboseCondition;
+import org.osgi.framework.Bundle;
+import org.osgi.framework.ServiceReference;
+
+/**
+ * A Utility-Class thats Provides static methods to create {@link Condition}s
+ * for an {@link Bundle}
+ */
+public interface BundleConditions {
```

Comment:
What are these static condition methods in an interface? That seems wrong. I don't see that we want anyone to "implement" these interfaces. Better that these classes are final classes with an empty private constructor.

Reply:
yes , changed to final class